src/Vezeeta.Infrastructure/Vezeeta.Infrastructure.csproj

```diff
@@ -7,6 +7,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.Identity.EntityFrameworkCore" Version="8.0.0" />
```

Comment:
You should remove this, as it is already referenced in `Domain Layer`. and this project references the `Domain Layer` so the `Microsoft.AspNetCore.Identity.EntityFrameworkCore` will be visible to it.